### DISCUSSION TITLE
Raspberry PI 400 test

### DISCUSSION THREAD

**Discussion by CleversonSA** [mbridak/not1mm#32] [General]
Hello,

Nice to see this project, I'll test it at my Raspberry Pi 400. I've sucessfully installed the app as described at READ.ME.

I'm not a very addicted to contests, but I want an app to register the QSO Calls, specially at FT8....in linux, specially for ARM architeture. So I've found your project :)

Regards, good QSOs, 73s

PU5TOS (https://www.qrz.com/db/PU5TOS)

**Comment by mbridak** [comment]
Hi, Thanks for trying it out. I'm not much of a contester either.
I just saw a hole to fill. Hope you find it useful.
